### ISSUE TITLE
Widespread bug causing a huge number of buffs to deceptively not stack with each other.

### ISSUE THREAD

**Issue by balakethelock** [slowtorta/turtlewow-bug-tracker#3994]
**Please, provide a clear description what the bug is:**
Very crappy "_IsExclusiveSpellAura" code in spellauras.cpp makes many buffs exclusive between one-another for no reason

**Steps to reproduce the behavior with as much detail as possible:**

1. Apply two affected auras at the same time
2. look at your character sheet or use  `.unit statinfo`
3. see that only the strongest aura is applying its effect

**IDs of affected creatures, items, quests or spells with a link to the relevant page:**
Too big to list, but there are many bug reports all directly caused by this #3437 #3686 #3356 #2436
Other things I personally tested, some ingame some in vmangos test server (It is the same, this code has not been touched for years)
- Wrath of cenarius proc, paladin T3 spell dam proc, and greater arcane elixir all don't stack with each other
- Visor of Command does not stack with strength elixir
- Shahram +50 stats buff does not stack with mark of the wild

Basically, think of any two buffs that give you the same stat, there is a huge chance they won't stack with each other and you'll never realize it.

**Expected behavior. Describe how it should work:**
In vanilla, buffs stacking with each other is the norm. Exclusivity between buffs is the exception. Instead of the code specifying a few exceptions that can stack, instead it should let everything stack except the ones we know shouldn't (like lesser firepower with greater firepower, agi elixir with mongoose, etc)

**Comment by balakethelock** [comment]
> Elxiir of brute force 18 stam doesn't stack with fortitude. Should be changed too, elixir is useless and I am not even sure if this is a bug or blizzlike but it's bad

That's intended behavior though, take it to forums and not github

**Comment by afizzler** [comment]
> Elxiir of brute force 18 stam doesn't stack with fortitude. Should be changed too, elixir is useless and I am not even sure if this is a bug or blizzlike but it's bad

It’s intended behavior, please go to forums instead of github
